crates/typst/src/introspection/locator.rs

```diff
@@ -23,14 +23,14 @@ use crate::introspection::{Introspector, Location};
 /// its locator. When a layouter needs to call multiple sublayouters, we need to
 /// make an explicit decision:
 ///
-/// - Split: When we're layouting multiple distinct children (or other pieces of
+/// - Split: When we're laying-out multiple distinct children (or other pieces of
```

Comment:
This is the change bit that I expect to be controversial (in a couple of parts).

`layout` is really a noun, the verb form is really `lay out`, see https://www.merriam-webster.com/dictionary/layout which notes the related formulations `laid out`; `laying out`; `lays out`. I've generally replaced things with `laying-out` although the hyphen isn't really desirable (I'd personally omit it).

I'm happy to drop this (or any other change).

Reply:
I'm aware that layout is not really a verb, but it's used across the whole codebase and not just in comments, but also in function names. I'd prefer to leave it as-is for now, as you've anticipated.